PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2656,7 +2656,7 @@ bool isUrlTextChar(TCHAR const c)
 		case '{':
 		case '}':
 		case '?':
-		case '\0x7f':
+		case '\u007F':

```

Comment:
@Uhf7 Was this an intention?
This line was addded in the commit https://github.com/notepad-plus-plus/notepad-plus-plus/commit/401bfcb0cd133d43f550dbe69275faf28663606b
It has decimal value `127`

Reply:
I presume the warning here is "duplicate case value"

Reply:
@sasumner No, the warning was `character constant too long for its type`.
As I understood, `'\0xNN'` format is intended only for ASCII char literals.

U007F is _DELETE_ symbol: https://www.fileformat.info/info/unicode/char/7f/index.htm
But I wanted to ensure if it was intended.

Reply:
GCC Warning:
```
../src/Notepad_plus.cpp:2659:8: warning: multi-character character constant [-Wmultichar]
   case '\0x7f':
        ^~~~~~~
```
This is not a valid character literal. If you slightly modify the code, and compile it using MSVC, it also gives you a warning.
The code:
```
#include <iostream>
int main()
{
    wchar_t c = '\0x7f';
    std::cout << c << '\n' << (int)c;
}
```
MSVC Warning:
```
1>e:\Development\cpp\etc\ConsoleTest\ConsoleApplication\Source.cpp(4,21): warning C4305: 'initializing': truncation from 'int' to 'char'
1>e:\Development\cpp\etc\ConsoleTest\ConsoleApplication\Source.cpp(4,14): warning C4309: 'initializing': truncation of constant value
```
Also, even without compiling:
![image](https://user-images.githubusercontent.com/9664141/122670356-5daab680-d1ca-11eb-9d0d-183c65c4bb79.png)

---

In this case, GCC is more demanding. And I think this is good since it helps us to spot a potential bug.